test/lsp/lsp_definition.ml

```diff
@@ -159,6 +159,47 @@ let%expect_test "simple-definition-requests-3" =
        7               05 Y PIC 9.
        8           PROCEDURE DIVISION. |}]
 
+(* This is wrong atm due to Lsp_position.sieve and item_definition with_loc interaction *)
+let%expect_test "definition-requests-condition" =
+  let { projdir; end_with_postproc }, server = make_lsp_project () in
+  print_definitions server ~projdir @@ extract_position_markers {cobol|
+        IDENTIFICATION DIVISION.
+        PROGRAM-ID. prog.
+        DATA DIVISION.
+        WORKING-STORAGE SECTION.
+        01 PIC X.
+          88 COND_|1-cond-in-def|_ VALUE "A".
+        PROCEDURE DIVISION.
+          SET _|1-cond-in-stmt|_COND TO TRUE
+          IF CO_|2-cond-in-stmt|_ND
+            DISPLAY "A"
+          END-IF
+          STOP RUN.
+    |cobol};
+  end_with_postproc [%expect.output];
+  [%expect {|
+    {"params":{"diagnostics":[],"uri":"file://__rootdir__/prog.cob"},"method":"textDocument/publishDiagnostics","jsonrpc":"2.0"}
+    1-cond-in-def (line 6, character 17):
+    No definition found
```

Comment:
Well spotted!

Reply:
It's the same error that causes this https://github.com/OCamlPro/superbol-studio-oss/pull/293#discussion_r1617345980 fyi

Reply:
Should I create an issue for this ?

Reply:
Yes indeed.

Reply:
https://github.com/OCamlPro/superbol-studio-oss/issues/297